### ISSUE TITLE
Cannot connect because of invalid ChannelType

### ISSUE THREAD

**Issue by xdpoi** [SpaceManiac/discord-rs#158]
Using user tokens (no idea why this does not affect bot tokens) cannot connect at all in current release because of the invalid ChannelType (for the channel categories). Thus user tokens cannot be used.

This is a pretty major bug, but is fixed in latest master, could you please do a release?

**Comment by Dentosal** [comment]
For me `master` branch is broken too. Minimal reproduction (at c6d89e5863341964017adb49c09d8637da162786):

```rust
extern crate discord;
use discord::Discord;

fn main() {
    let token = "ACCESS TOKEN HERE";
    let dc = Discord::from_bot_token(token.as_str()).unwrap();
    dc.get_servers().unwrap(); // panics
}
```

Error contains:

    Message("invalid type: string \"$REMOVED_INTEGER\", expected a u64 or parseable string")

**Comment by Dentosal** [comment]
This seems to be fixed in fork [tinaun/discord-rs](https://github.com/tinaun/discord-rs).

**Comment by SpaceManiac** [comment]
Fixed as of 3976cbe31c8bf99a8986f81e16c9cb09df3e2fa4
